apps/pie-docs/src/assets/js/show-more.js

```diff
@@ -0,0 +1,59 @@
+let shown = false;
+let showMoreButtons;
+
+window.addEventListener('DOMContentLoaded', () => {
+    showMoreButtons = document.querySelectorAll('[data-js="show-more"]');
+    const componentStatusTables = document.querySelectorAll('[class="c-componentStatusTable "]');
+
+    if (!showMoreButtons) return;
+    if (!componentStatusTables) return;
```

Comment:
both values are `nodeList` so we need to check the length here cause empty list is a truthy value. 

```suggestion
    if (!showMoreButtons.length || componentStatusTables.length ) return;
```

Reply:
🤦‍♀️ thank youu!